azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/CallCompositeActivity.kt

```diff
@@ -75,6 +76,13 @@ internal class CallCompositeActivity : AppCompatActivity() {
                 configuration.themeConfig?.theme!!, true
             )
         }
+
+        configuration.localizationConfig?.let { localeConfig ->
+            Locale.setDefault(Locale(localeConfig.language))
```

Comment:
What happens here is value of localeConfig.language is not a valid Locale?

Reply:
If it is not valid(for example "arr"), it will show English String